rdr_service/model/bq_genomics.py

```diff
@@ -208,6 +208,54 @@ class BQGenomicJobRunView(BQView):
     __table__ = BQGenomicJobRun
 
 
+class BQGenomicFileProcessedSchema(BQSchema):
+    id = BQField('id', BQFieldTypeEnum.INTEGER, BQFieldModeEnum.REQUIRED)
+    created = BQField('created', BQFieldTypeEnum.DATETIME, BQFieldModeEnum.REQUIRED)
+    modified = BQField('modified', BQFieldTypeEnum.DATETIME, BQFieldModeEnum.REQUIRED)
+
+    # PDR-149:  Need to preserve RDR table id values
+    run_status = BQField('run_status', BQFieldTypeEnum.STRING, BQFieldModeEnum.NULLABLE,
```

Comment:
I'm unclear on these fields (run_status through result_message) and how the BQ schemas are going to mirror RDR.   These are already defined in the BQGenomicJobRun schema and seem to relate to the RDR genomic_job_run table.   Are they being replicated here?

Reply:
No, that is a mistake. Thanks for pointing that out. I will remove those fields.